api/src/main/resources/workbench-api.yaml

```diff
@@ -5855,6 +5863,23 @@ definitions:
     properties:
       runtime:
         "$ref": "#/definitions/Runtime"
+  PersistentDiskRequest:
+    description: configuration to create a disk
+    type: object
+    required:
+      - name
```

Comment:
is size should also be required?

Reply:
Not really I think. Because we could just pass a name to specify an existing PD.